src/components/GoalSetting/GoalSettingContainer.vue

```diff
@@ -0,0 +1,293 @@
+<template>
+	<div class="tw-min-h-screen lg:tw-min-h-full tw-mt-5">
+		<button
+			class="tw-flex tw-gap-1 tw-items-center tw-font-medium tw-mt-3 tw-mb-4"
+		>
+			<KvMaterialIcon
+				:icon="mdiChevronLeft"
+				class="tw-ml-0.5"
+			/>
+			To dashboard
```

Comment:
This button currently doesn't have an action or analytics. Looks like this should return to MyKiva and the event is listed in the ticket.

Reply:
True, I added before knowing where is returning. Done.